### PR TITLE
Various cosmetic tweaks to the cloud package

### PR DESCRIPTION
**Pull request by ejj**

### CODE REVIEW

File: cloud/amazon/amazon.go [hunk 1]
```diff
@@ -103,8 +103,8 @@ type bootReq struct {
 	preemptible bool
 }
 
-// Boot creates instances in the `prvdr` configured according to the `bootSet`.
-func (prvdr *Provider) Boot(bootSet []machine.Machine) error {
+// Boot creates instances in the `client` configured according to the `bootSet`.
```
Comment (kklin) [inline cloud/amazon/amazon.go]: Why rename `prvdr` to `client` here?

File: cloud/amazon/amazon.go [hunk 2]
```diff
@@ -202,14 +202,14 @@ func (prvdr *Provider) bootSpot(br bootReq, count int64) error {
 	return err
 }
 
-// Stop shuts down `machines` in `prvdr`.
-func (prvdr *Provider) Stop(machines []machine.Machine) error {
+// Stop shuts down `machines` in `client.
```
Comment (kklin) [inline cloud/amazon/amazon.go]: Missing the closing backtick here, and same question about renaming `prvdr` to `client`.

File: cloud/amazon/amazon.go [hunk 3]
```diff
@@ -361,8 +361,8 @@ func (prvdr *Provider) listInstances() (instances []awsMachine, err error) {
 	return instances, nil
 }
 
-// List queries `prvdr` for the list of booted machines.
-func (prvdr *Provider) List() (machines []machine.Machine, err error) {
+// List queries `client` for the list of booted machines.
```
Comment (kklin) [inline cloud/amazon/amazon.go]: Same here re rename

File: cloud/cloud.go [hunk 4]
```diff
@@ -217,7 +216,7 @@ func (cld cloud) updateCloud(machines []joinMachine, fn machineAction, action st
 }
 
 type joinMachine struct {
```
Comment (kklin) [inline cloud/cloud.go]: This struct isn't really necessary now, but I suspect you're ditching it later in your rewrite anyways

Comment (ejj) [inline cloud/cloud.go]: yeah, being ditched later

File: Makefile [hunk 5]
```diff
@@ -3,7 +3,7 @@ PACKAGES=$(shell govendor list -no-status +local)
 NOVENDOR=$(shell find . -path -prune -o -path ./vendor -prune -o -name '*.go' -print)
 LINE_LENGTH_EXCLUDE=./api/pb/pb.pb.go \
 		    ./cloud/amazon/client/mocks/% \
-		    ./cloud/cloudcfg/template.go \
+		    ./cloud/cfg/template.go \
```
Comment (ejj) [inline Makefile]: 👍

File: cloud/amazon/amazon.go (hunk 1 above)
Comment (ejj) [inline cloud/amazon/amazon.go]: rebase error.

**Pull request merged**